ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -457,6 +475,42 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
                   </ButtonLink>
                 </Box>
               )}
+              {promptToCreateSolanaAccount && !solanaAccountCreated && (
+                <Box
+                  display={Display.Flex}
+                  flexDirection={FlexDirection.Column}
+                  justifyContent={JustifyContent.center}
+                  alignItems={AlignItems.center}
+                  marginTop={4}
+                  gap={2}
+                >
+                  <Text
+                    variant={TextVariant.bodyMd}
+                    color={TextColor.textAlternative}
+                    textAlign={TextAlign.Center}
+                  >
+                    {selectedAccounts.length === 0
+                      ? t('solanaAccountRequired')
+                      : t('solanaAccountRequested')}
```

Comment:
won't `selectedAccounts.length` always be zero in this block of components?

Reply:
ohhhh... my wires are crossed. For some reason i thought this prompt would only be set by the wallet_createSession handler if solana was the **only** scope selected